### PR TITLE
Fix memory databases for sqlite

### PR DESCRIPTION
**Pull request by TimDiekmann**
r? @wencke-lm

### CODE REVIEW

File: slurk/extensions/database.py [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 Base = declarative_base()
 
 
+@event.listens_for(Engine, "connect")
+def set_sqlite_pragma(dbapi_connection, connection_record):
+    cursor = dbapi_connection.cursor()
+    cursor.execute("PRAGMA foreign_keys=ON")
+    cursor.close()
+
+
```
Comment (TimDiekmann) [inline slurk/extensions/database.py:15]:
```suggestion
```

File: slurk/extensions/database.py [hunk 2]
```diff
@@ -24,23 +33,30 @@ def engine(self):
...
+            self._connect_args = {"check_same_thread": False}
+            self._poolclass = StaticPool
+
     def bind(self, engine):
         self._engine = engine
-        if "sqlite" in engine.url:
-            from logging import getLogger
+        self._session.configure(bind=engine)
+
```
Comment (TimDiekmann) [inline slurk/extensions/database.py:47]:
```suggestion
```

**Review by wencke-lm — COMMENTED** [review]
[Edit] Receiving layouts works now, for whatever reason it didn't before o.O need to test some more.

File: scripts/start_server.sh [hunk 3]
```diff
@@ -36,20 +30,10 @@ else
...
-    else
-        param="-e SLURK_DATABASE_URI=$SLURK_DATABASE_URI"
-    fi
-
     docker run -d \
         --name=$SLURK_DOCKER \
         -p $PORT:80 \
         -e SLURK_SECRET_KEY=$SLURK_SECRET_KEY \
         -e SLURK_DISABLE_ETAG=${SLURK_DISABLE_ETAG:-False} \
-        -e FLASK_ENV=$FLASK_ENV \
```
[the remark this answers is not in the document]
Comment (TimDiekmann) [reply scripts/start_server.sh:52]: That was by mistake, thanks!

**Review by wencke-lm — APPROVED** [review]

**Pull request merged**